docs/tutorials/first_steps/08_create_workflow.rst

```diff
@@ -251,7 +278,7 @@ We can now use ``renku run`` to generate the second step of our workflow:
 
 .. code-block:: console
 
-    $ renku run python src/count_flights.py data/output/flights-filtered.csv data/output/flights-count.txt
+    $ renku run X src/CountFlights.* data/output/flights-filtered.csv data/output/flights-count.txt
```

Comment:
See comment above about using `src/{python,julia,r}` folders.

Reply:
Opted for `src` folder for the tutorial, with note on additional organisation.